src/main/java/service/Parser.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface Parser {
+    List<FruitTransaction> parse(List<String> dataFromFile);
```

Comment:
```suggestion
    List<FruitTransaction> parse(List<String> lines);
```
Tomorrow you will receive lines by email. Do we need to refactor this interface in that case?